data_architect_misc/data_transformer/configs/AED_Russia/config.json

```diff
@@ -64,12 +64,12 @@
       {
         "__function_comment__": "Create HARMONIZED_YEAR column from existing YEAR column name.",
         "function_name": "add_HARMONIZED_YEAR_column_by_renaming_existing_column",
-        "function_args": ["YEAR"]
+        "function_args": [ "YEAR" ]
```

Comment:
Don Maicol, 

I see that you have added extra space characters BEFORE and AFTER the list items. Maybe it's because you are using a different code editor and that code editor is not following Python's standard/recommended style guide  [https://www.python.org/dev/peps/pep-0008/]. If you are using something other than PyCharm, please make sure to set your linter in that software to PEP8 or something recent. Otherwise, you'll end up adding these extra spaces, and although it's not going to cause issues when running the code, it's better to follow convention in languages like Python that are a bit sensitive about spaces and tabs.

Hope that makes sense.